File: viasoft.licensing.licensingmanagement/Viasoft.Licensing.LicensingManagement.Domain/Services/TenantManagement/Dto/Environment/OrganizationUnitEnvironment.cs
using System;
using Viasoft.Core.DDD.Entities;
using Viasoft.Data.Attributes;

namespace Viasoft.Licensing.LicensingManagement.Domain.Services.TenantManagement.Dto.Environment
{
    public class OrganizationUnitEnvironment : Entity
    {
        [StrictRequired]
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsProduction { get; set; }
        public bool IsMobile { get; set; }
        public bool IsWeb { get; set; }
        public bool IsDesktop { get; set; }
        public string DatabaseName { get; set; }
        public Guid OrganizationUnitId { get; set; }
        public bool IsActive { get; set; }
        public string DesktopDatabaseVersion { get; set; }
    }
}

File: viasoft.licensing.licensingmanagement-frontend/libs/licensing-licensingmanagement-licensedtenant/src/lib/clients/license-management/model/organizationUnitEnvironment.ts
export interface OrganizationUnitEnvironment { 
    name: string;
    description?: string | null;
    isProduction?: boolean;
    isMobile?: boolean;
    isWeb?: boolean;
    isDesktop?: boolean;
    databaseName?: string | null;
    organizationUnitId?: string;
    isActive?: boolean;
    id?: string;
}
